pkg/service/backup/backup.go

```diff
@@ -3,59 +3,12 @@
 package backup
 
 import (
-	"strings"
-
 	"github.com/pkg/errors"
-	"github.com/scylladb/go-set/strset"
 	"github.com/scylladb/scylla-manager/v3/pkg/scyllaclient"
 	. "github.com/scylladb/scylla-manager/v3/pkg/service/backup/backupspec"
-	"github.com/scylladb/scylla-manager/v3/pkg/util/slice"
 	"go.uber.org/multierr"
 )
 
-func checkDCs(dcAtPos func(int) (string, string), n int, dcMap map[string][]string) (err error) {
-	allDCs := strset.New()
-	for dc := range dcMap {
-		allDCs.Add(dc)
-	}
-
-	for i := 0; i < n; i++ {
-		dc, str := dcAtPos(i)
-		if dc == "" {
-			continue
-		}
-		if !allDCs.Has(dc) {
-			err = multierr.Append(err, errors.Errorf("%q no such datacenter %s", str, dc))
-		}
-	}
-	return
-}
```

Comment:
All this logic is moved to dclimit.go, right ?

Reply:
Yes - the only difference is that the error messages were slightly changed due to functions being more general.